tools/debian/control

```diff
@@ -24,6 +24,7 @@ Build-Depends: debhelper (>= 10),
                glib-networking,
                openssh-client <!nocheck>,
                python3,
+               dbus,
```

Comment:
This change is not explained by the commit message.

Reply:
And apparently we never had it in our debian image, so reverted.